templates/flavors/base/cluster.yaml

```diff
@@ -1,19 +1,18 @@
+---
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: Cluster
 metadata:
   name: ${CLUSTER_NAME}
   labels:
-    cni: cilium
```

Comment:
I think we should leave these on here since they're currently needed for the clusters to work